make/build.sh

```diff
@@ -564,14 +549,14 @@ setup_junit() {
 
     if [ -z "${JUNIT_JAR_URL:-}" ]; then
         if [ -n "${JUNIT_JAR_URL_BASE:-}" ]; then
-            JUNIT_JAR_URL="${JUNIT_JAR_URL_BASE}/junit/junit/${JUNIT_VERSION}/junit-${JUNIT_VERSION}.jar"
+            JUNIT_JAR_URL="${JUNIT_JAR_URL_BASE}/org/junit/platform/junit-platform-console-standalone/${JUNIT_VERSION}/junit-platform-console-standalone-${JUNIT_VERSION}.jar"
         fi
     fi
 
     local JUNIT_DEPS_DIR="${DEPS_DIR}/junit"
 
     if [ -n "${JUNIT_JAR_URL:-}" ]; then
-        JUNIT_JAR="${JUNIT_DEPS_DIR}/$(basename ${JUNIT_JAR_URL})"
+        JUNIT_JAR="${JUNIT_DEPS_DIR}/junit.jar"
```

Comment:
It would be nicer to retain the full base name of the jar file if possible

Reply:
Done via https://github.com/openjdk/jtreg/pull/63/commits/93a391a5213f898caebe83611643921ead6d94a4